tests/test_opt.py

```diff
@@ -190,6 +194,38 @@ def test_fan_in(self):
         self.assertEqual(len(new_nodes), 7)
         self.assertIsNotNone(model)
 
+    def test_opt_on_model(self):
```

Comment:
which case this unit test try to handle?

Reply:
the goal of this test is to cover optimize_onnx_model, which was missed in any test.